.github/workflows/regression-test.yml

```
@@ -0,0 +1,44 @@
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Install Dependencies
+        run: npm ci
+
+      - name: Run Cypress Tests
+        uses: cypress-io/github-action@v6
+        with:
+          start: npm start # or whatever starts your app
+          wait-on: 'http://localhost:3000' 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)